pkg/provider/terraform/instance/plugin.go

```diff
@@ -52,6 +52,9 @@ const (
 
 	// scopeGlobal is the scope key for global resources
 	scopeGlobal = "global"
+
+	// NameTag is the name of the tag that contains the instance name
+	NameTag = "instance.name"
```

Comment:
Can we prefix this with `infrakit.` so that when users look at the resources and their tags they can tell this is an infrakit managed resource?

Reply:
Sure, it'll be:
```
NameTag = "infrakit.instance.name"
```
Thanks for the review!